apps/api/src/domain/models/audio-item/commands/transcripts/import-line-items-to-transcript/import-line-items-to-transcript.command-integration.spec.ts

```diff
@@ -0,0 +1,137 @@
+import generateDatabaseNameForTestSuite from '../../../../../../persistence/repositories/__tests__/generateDatabaseNameForTestSuite';
+import formatAggregateType from '../../../../../../view-models/presentation/formatAggregateType';
+import { assertCommandSuccess } from '../../../../__tests__/command-helpers/assert-command-success';
+import { CommandAssertionDependencies } from '../../../../__tests__/command-helpers/types/CommandAssertionDependencies';
+import buildDummyUuid from '../../../../__tests__/utilities/buildDummyUuid';
+import { AudioItem } from '../../../entities/audio-item.entity';
+import { Video } from '../../../entities/video.entity';
+import { ImportLineItemsToTranscript } from './import-line-items-to-transcript.command';
+
+const commandType = `IMPORT_LINE_ITEMS_TO_TRANSCRIPT`;
+
+const testDatabaseName = generateDatabaseNameForTestSuite();
+
+const systemUserId = buildDummyUuid();
+
+const existingAudioItem = getValidAggregateInstanceForTest(AggregateType.audioItem);
+
+const existingVideo = getValidAggregateInstanceForTest(AggregateType.video);
+
+const validAudiovisualItems = [existingAudioItem, existingVideo];
+
+describe(commandType, () => {
+    let testRepositoryProvider: TestRepositoryProvider;
+
+    let commandHandlerService: CommandHandlerService;
+
+    let app: INestApplication;
+
+    let idManager: IIdManager;
+
+    let assertionHelperDependencies: CommandAssertionDependencies;
+
+    beforeAll(async () => {
+        ({ testRepositoryProvider, commandHandlerService, idManager, app } =
+            await setUpIntegrationTest({
+                ARANGO_DB_NAME: testDatabaseName,
+            }));
+
+        assertionHelperDependencies = {
+            testRepositoryProvider,
+            commandHandlerService,
+            idManager,
+        };
+    });
+
+    afterAll(async () => {
+        await app.close();
+    });
+
+    beforeEach(async () => {
+        await testRepositoryProvider.testSetup();
+    });
+
+    afterEach(async () => {
+        await testRepositoryProvider.testTeardown();
+    });
+
+    validAudiovisualItems.forEach((validInstance) => {
+        describe(`when transcribing a ${formatAggregateType(validInstance.type)}`, () => {
+            const [startTime, endTime] = validInstance.getTimeBounds();
+
+            const audioLength = endTime - startTime;
+
+            const numberOfTimestampsToGenerate = 10;
+
+            const epsilon = 0.0001;
+
+            const allTimestamps = Array(numberOfTimestampsToGenerate)
+                .fill(0)
+                .map((_, index) => [
+                    startTime + epsilon + (index * audioLength) / numberOfTimestampsToGenerate,
+                    startTime +
+                        ((index + 1) * audioLength) / numberOfTimestampsToGenerate -
+                        epsilon,
+                ]);
+
+            const dummyText = 'bla bla bla';
```

Comment:
We now want to add many line items. So we'll need a way to build dummy text for a given time stamp. This might help:
```ts
const buildDummyText = ([inPoint,outPoint]: [number,number]) => `text for line item with time range: [${inPoint},${outPoint}]`,
```

Reply:
this was done